src/history.h

```diff
@@ -40,9 +40,13 @@ struct io_streams_t;
 typedef std::vector<wcstring> path_list_t;
 
 enum history_search_type_t {
-    // The history searches for strings containing the given string.
+    // We don't know what type of search to perform.
+    HISTORY_SEARCH_TYPE_UNKNOWN,
```

Comment:
Thanks, fix looks good. Incidentally I have no objection if you want to start enums at 1 instead of 0.